site/es/_data/home.yml

```diff
@@ -81,15 +81,15 @@ stats:
       modifiers:
         shape: circle
     - stat: 50M
-      desc: Students and educators using Chromebooks worldwide
+      desc: Estudiantes y educadores usando Chromebooks alrededor del mundo
       modifiers:
         shape: triangle
         scale: half
 caseStudies:
-  title: Get inspired.
+  title: Encuentra inspiración.
   subtitle: Explore how brands are unlocking more possibilities with Chrome OS.
   cta:
-    text: All Case Studies
+    text: Casos de estudio
     url: stories
 m100:
```

Comment:
Can you please take a look at this? [This](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L97-L107) is the current content for it